### PR TITLE
[9.0] [MIG] Backport of slow_statement_logger from 12.0

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
Backport of #1729

### CODE REVIEW

File: slow_statement_logger/__init__.py [hunk 1]
```diff
@@ -7,13 +7,14 @@
...
-from odoo.tools import config
+from openerp import sql_db
+from openerp.tools import config
 
 _logger = logging.getLogger(__name__)
 
 # TODO Odoo option? server_environment?
-LOG_MIN_DURATION_STATEMENT = int(config.get("log_min_duration_statement", "-1"))
+LOG_MIN_DURATION_STATEMENT = int(
+    config.get("log_min_duration_statement", "-1"))
```
Comment (sbidoul) [inline slow_statement_logger/__init__.py]: @ThomasBinsfeld out of curiosity, is there a reason to wrap this line? Normally MQT accepts 88 characters on all branches now.

Comment (ThomasBinsfeld) [reply slow_statement_logger/__init__.py]: Oops, i thought it was only on 13.0 branch. Fixed

**Comment by sbidoul** [comment]
@ThomasBinsfeld we need to remove the `static` directory to avoid the module to be autoimported by Odoo. There is also an OCA specific key to add in the manifest to avoid index.html and icon generation (I don't remember it precisely right now, sorry).

**Comment by ThomasBinsfeld** [comment]
> @ThomasBinsfeld we need to remove the `static` directory to avoid the module to be autoimported by Odoo. There is also an OCA specific key to add in the manifest to avoid index.html and icon generation (I don't remember it precisely right now, sorry).

Done. The key in the manifest is `"preloadable": False,`.

**Review by sbidoul — APPROVED** [review]
Thanks! The travis error seems unrelated and I suppose nobody is interested to make this old branch green again.

**Review by houssine78 — APPROVED** [review]
LGTM

**Review by remytms — APPROVED** [review]
Seams OK. :)

**Comment by dreispt** [comment]
Build is red, please fix

**Pull request closed without merging**